nixos/modules/services/mail/postfix.nix

```diff
@@ -900,4 +890,13 @@ in
       services.postfix.mapFiles.client_access = checkClientAccessFile;
     })
   ]);
+
+  imports = [
+   (mkRemovedOptionModule [ "services" "postfix" "sslCACert" ]
+     "services.postfix.sslCACert was replaced by services.postfix.clientTlsCAFile. In case you intend to add trusted server CA use services.postfix.config.")
```

Comment:
```suggestion
     "services.postfix.sslCACert was replaced by services.postfix.clientTlsCAFile. In case you intend to add trusted server CA use services.postfix.extraConfig.")
```